common/src/main/cpp/classes/openxr_fb_scene_manager.cpp

```diff
@@ -300,12 +300,12 @@ void OpenXRFbSceneManager::_create_scene_anchor(const Ref<OpenXRFbSpatialEntity>
 }
 
 Ref<PackedScene> OpenXRFbSceneManager::get_scene_for_entity(const Ref<OpenXRFbSpatialEntity> &p_entity) const {
-	Array semantic_labels = p_entity->get_semantic_labels();
+	PackedStringArray semantic_labels = p_entity->get_semantic_labels();
```

Comment:
Can we instead return `return String(label_data).to_lower().split(",");` in `OpenXRFbSceneExtensionWrapper::get_supported_semantic_labels();` to have the semantic labels in lower case all the time.

Switching back and forth between `to_lower` and `to_upper` in different sections of the code is likely to introduce accidental subtle bugs.

Reply:
Sure, we could do that.

My original plan was to keep them in their original upper-cased form, because that's what we get from OpenXR. However, I was surprised to learn that Godot automatically lower-cases properties at some point (I think when saving to the .tscn file?), and so that didn't actually work. :-)

So, I attempted to switch to (1) using lower-case for the scene property names, but (2) upper-case when returning the list of semantic labels (so they matched the spec). But I clearly missed a couple spots that needed to change!

Anyway, if we're cool with just telling devs that we always deal with the semantic labels in lower-case (despite the spec giving them in upper-case), we could do that

Reply:
> Anyway, if we're cool with just telling devs that we always deal with the semantic labels in lower-case (despite the spec giving them in upper-case), we could do that

That sounds like a reasonable approach to me!

On the topic of documentation, do you know if GDExtension provides similar documentation mechanism as the core nodes? It'd be handy for developers to be able to access the documentation straight from the editor.

Reply:
> On the topic of documentation, do you know if GDExtension provides similar documentation mechanism as the core nodes?

There will be a mechanism in Godot 4.3! However, at this moment, it isn't implemented in godot-cpp yet (I have a draft PR that I need to find the time to finish :-))

Reply:
Sweet! 
Would that address https://github.com/GodotVR/godot_openxr_vendors/issues/45?

Reply:
Maybe. It basically allows us to give the editor the same kind of XML docs as we can in Godot itself. But export settings are treated as properties of the `EditorExportPlatform` - I don't know if there's a way for us to pass in some XML that would be added to that? I'd need to investigate a bit

Reply:
I've updated everywhere in the API where we expose semantic labels to return them in lower-case. This required some changes in the demo. It feels a little weird not matching the names from the spec, but it works!